client/src/templates/Challenges/utils/frame.ts

```diff
@@ -124,7 +129,8 @@ const createFrame =
     const frame = document.createElement('iframe');
     frame.id = id;
     if (typeof title === 'string') {
-      frame.title = title;
+      frame.title = i18next.t('misc.iframe-preview', { title });
+      frame.lang = getLangCode(clientLocale);
```

Comment:
This really does not make sense to me.

The `iframe` only shows English content. So, to tell a browser/screen-reader it should expect anything else is misleading no?

Reply:
Yes we should probably set the `lang` attribute to `en` here.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient